### PR TITLE
Ncit

### LINKED ISSUE #13: Implement NCIt normalizer
It would be great to get NCIt concept codes where possible. For drugs, this will be descendent concepts of [Pharmacologic Substance](https://ncithesaurus.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C1909&ns=ncit).

### PR DESCRIPTION
**Pull request by jsstevenson**
Closes #13

### CODE REVIEW

File: therapy/schemas.py [hunk 1]
```diff
@@ -122,7 +123,8 @@ class SourceIDAfterNamespace(Enum):
     WIKIDATA = "Q"
     CHEMBL = "CHEMBL"
     DRUGBANK = "DB"
-    CASREGISTRY = ""
+    NCIT = "C"
+    CASREGISTRY = ""  # do we need these for sources that we don't query?
```
Comment (korikuzma) [inline therapy/schemas.py]:
```suggestion
    CASREGISTRY = "" 
```

File: therapy/etl/ncit.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            class_object = ncit[result[0].toPython().split('#')[1]]
+            uq_nodes.add(class_object)
+        return uq_nodes
+
+    def _add_meta(self, db: Session):
+        meta_object = Meta(src_name=SourceName.NCIT.value,
+                           data_license="CC BY 4.0",
+                           data_license_url="https://creativecommons.org/licenses/by/4.0/legalcode",  # noqa F401
+                           version=self._version,
+                           data_url="https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/20.09d_Release/Thesaurus_20.09d.OWL.zip",)  # noqa F401
```
Comment (korikuzma) [inline therapy/etl/ncit.py:126]:
```suggestion
                           data_url="https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/20.09d_Release/",)  # noqa F401
```
In ChEMBL, we gave the index to the page that contains the db. We might want to do this for ncit?

**Review by korikuzma — APPROVED** [review]

File: therapy/schemas.py (hunk 1 above)
Comment (jsstevenson) [reply therapy/schemas.py]: done!

File: therapy/etl/ncit.py (hunk 2 above)
Comment (jsstevenson) [reply therapy/etl/ncit.py]: done!

**Pull request merged**